packages/flutter_tools/lib/src/localizations/gen_l10n.dart

```diff
@@ -715,19 +694,15 @@ class LocalizationsGenerator {
   /// string format.
   final Map<File, String> _languageFileMap = <File, String>{};
 
-  /// Contains the generated application's localizations and localizations delegate
-  /// classes.
-  String _generatedLocalizationsFile;
```

Comment:
This was created and used in the same method, so I removed it from the instance.